internal/integration/influxdb/influxdb.go

```diff
@@ -195,6 +195,9 @@ func (i *Integration) HandleUplinkEvent(ctx context.Context, _ models.Integratio
 		Values: map[string]interface{}{
 			"value": 1,
 			"f_cnt": pl.FCnt,
+            "adr": pl.ADR,
+            "data_rate": pl.DR,
```

Comment:
I'm not sure if the data-rate should be stored as a value, please see my comment in #561.